packages/devtools/src/launcher.ts

```diff
@@ -89,12 +97,15 @@ async function launchChrome (capabilities: ExtendedCapabilities) {
         chromeFlags.push(`--user-agent=${mobileEmulation.userAgent}`)
     }
 
+    if (port)
+        log.info(`Requesting to connect to Google Chrome on port: ${port}`)
     log.info(`Launch Google Chrome with flags: ${chromeFlags.join(' ')}`)
 
     const chrome = await launchChromeBrowser({
         chromePath: chromeOptions.binary,
         ignoreDefaultFlags: true,
-        chromeFlags
+        chromeFlags,
+        port
```

Comment:
So we don't pass along an undefined property.
```suggestion
        ...(port ? { port } : {})
```

Reply:
One thing I noticed is the GitHub Actions tests are failing in jamesmortensen:main:  https://github.com/jamesmortensen/webdriverio/runs/1910557241?check_suite_focus=true

I haven't had a chance to take a look yet, but if you see why I'm getting these, I would appreciate any guidance.  

I'll answer the other comments later today.  Thank you.

Reply:
I would try updating your branch with current `main` branch.

Reply:
I modified the code to not pass in undefined as suggested, although we should note that before introducing the port property, the chrome-launcher was receiving this property as undefined.